Sources/PermissionsSwiftUINotification/JMNotificationPermissionManager.swift

```diff
@@ -57,7 +57,7 @@ public final class JMNotificationPermissionManager: PermissionType.PermissionMan
     var notificationManager = UNUserNotificationCenter.current()
 
     override public func requestPermission(completion: @escaping (Bool, Error?) -> Void) {
-        notificationManager.requestAuthorization(options: [.badge,.alert,.sound]){ granted, error in
+        notificationManager.requestAuthorization(options: [.badge,.alert,.sound, .criticalAlert]){ granted, error in
```

Comment:
Hmm, @Roberdan maybe I'm missing something, but blindly requesting the `.criticalAlert` seems a little extraneous. I'd say 99% of use case don't need the critical alert option. Especially when Apple's docs say:
> When [UNNotificationSetting.enabled](https://developer.apple.com/documentation/usernotifications/unnotificationsetting/enabled), this property authorizes the app to play critical [sound](https://developer.apple.com/documentation/usernotifications/unnotificationcontent/1649871-sound)s that ignore Do Not Disturb and the device’s mute switch.
Critical alerts require a special entitlement issued by Apple.


I like the idea, maybe add a parameter in the public API to make this configurable? @Roberdan